migrations/2022-11-14-194539_create_attributes/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE nft_attributes (
+  nft_metadata_id  TEXT NOT NULL,
+  nft_contract_id TEXT NOT NULL,
+  attribute_type TEXT NOT NULL,
```

Comment:
This should be named `attribute_trait_type`, while we are lacking `attribute_display_type` (https://docs.opensea.io/docs/metadata-standards#attributes)

Reply:
As discussed for now I think it might be good to keep the name generic as attribute because the point of this is searchability and we will certainly want to add other fields from the reference blob to this table down the line.